apps/LineRider/install

```diff
@@ -2,40 +2,40 @@
 
 #Bot automatically updates this when I make a new release.
 version=2023.1.1
-#Buster version
-version2=2022.5.1
 
 #Install Depends
 install_packages apt-transport-https dirmngr gnupg ffmpeg mono-devel unzip zip || exit 1
 
 mkdir -p ~/LineRider
 cd ~/LineRider || error "Could not move to folder"
 
-if [[ $(get_codename) == "buster" ]]; then
-  use_version="$version2"
-else
-  use_version="$version"
-fi
-
 #Download LRA-Community-Edition build from repo
 status "Downloading LineRider... "
-wget -O linerider.zip https://github.com/Sussy-OS/LRA-Community-Edition/releases/download/${use_version}/LineRider.game.auto-release.${use_version}-LR.zip || error "Could not wget LineRider from GitHub!"
+wget -O linerider.zip https://github.com/Sussy-OS/LRA-Community-Edition/releases/download/${version}/LineRider.game.auto-release.${version}-LR.zip || error "Could not wget LineRider from GitHub!"
 
 #Install
-unzip linerider.zip || error "Could not unzip the files"
-rm linerider.zip
+unzip linerider.zip || error "Could not unzip linerider.zip!"
+rm linerider.zip || error "Failed to delete linerider.zip!"
+cd ~/
+sudo mv -f ~/LineRider /opt/LineRider || error "Failed to move source folder to /opt!"
+sudo chmod +x /opt/LineRider/linerider.exe
```

Comment:
it is better to work out of the `/tmp` folder when doing temporary operations. that way if anything fails or you don't clean something up, nothing is left in the users home folder

Reply:
Okay, will change in a moment.

Reply:
Done